### PR TITLE
 OPSEXP-1932 Update architecture image with Admin Console

### PR DESCRIPTION
**Pull request by upandey123**
The new Enterprise trial of ACS (7.3) includes the Admin Console. The diagram ([compose-enterprise.png](https://docs.alfresco.com/content-services/images/compose-enterprise.png)) in the install using Docker containers documentation ([Alfresco Docs - Install using Docker Compose](https://docs.alfresco.com/content-services/latest/install/containers/docker-compose/#deployment-project-in-github) ) needs to be updated to include the Admin Console.

Ref: OPSEXP-1932

### CODE REVIEW

**Review by gionn — COMMENTED** [review]
LGTM but wait also for alex

**Review by alxgomz — CHANGES_REQUESTED** [review]
Please also commit the source of the diagram

**Comment by alxgomz** [comment]
Your style looks neat compared to previous one. Can you also regenerate the other docker-compose diagrams so they look they have the same look & feel?

**Comment by upandey123** [comment]
> Please also commit the source of the diagram

Have gone through this document the Detailed Architecture of the workflow.

https://docs.alfresco.com/content-services/latest/develop/software-architecture/#detailed-architecture

**Comment by upandey123** [comment]
> Your style looks neat compared to previous one. Can you also regenerate the other docker-compose diagrams so they look they have the same look & feel?

These are the other docker--compose diagrams I have got.

Screenshot 2023-02-02 at 10 27 15 PM

Screenshot 2023-02-02 at 10 26 53 PM

**Comment by upandey123** [comment]
Have created a new task for other diagrams to update and will perform it. If this seems good kindly approve!

File: README.md [hunk 1]
```diff
@@ -143,3 +143,15 @@ git push origin vx.x.x-SNAPSHOT --force
 ```
 
 Once the tagged workflow is successful, the release process has completed.
+
+## How to update workflow diagrams of Alfresco latest version
+
+* Go to the `docs/docker-compose/diagrams` folder of the repository and there you will find 2 plantuml source files and 2 diagram files.
```
Comment (alxgomz) [inline README.md:149]:
```suggestion
* Go to the `docs/docker-compose/diagrams` and/or `docs/helm/diagrams` folders of the repository and there you will find 2 plantuml source files and their PNG diagram files.
```

File: docs/docker-compose/diagrams/docker-compose-enterprise.puml [hunk 2]
```diff
@@ -30,23 +31,26 @@ node "Docker Compose" {
 
 Client --> proxy
 
-proxy --> acs
+proxy --> aac
 proxy --> share
 proxy --> adw
+proxy --> acs
 
+adw --> acs
```
Comment (alxgomz) [inline docs/docker-compose/diagrams/docker-compose-enterprise.puml:39]: This one should not be there. Please fix and regenerate

**Review by alxgomz — DISMISSED** [review]

**Review by alxgomz — APPROVED** [review]

**Pull request merged**